doc/devel/documenting_mpl.rst

```diff
@@ -39,50 +39,46 @@ Sphinx_.
 
 .. note::
 
-   An exception to this are the directories :file:`gallery` and
+   An exception to this are the directories :file:`examples` and
    :file:`tutorials`, which exist in the root directory.  These contain Python
    files that are built by `Sphinx Gallery`_.  When the docs are built,
-   directories of the same name will be generated inside of :file:`docs/`.  The
-   generated directories :file:`docs/gallery` and :file:`docs/tutorials` can be
-   safely deleted.
+   the directories :file:`docs/gallery` and :file:`docs/tutorials`
+   are generated.
 
-The configuration file for Sphinx is :file:`doc/conf.py`.  It controls which
+The configuration file for Sphinx is :file:`doc/conf.py`. It controls which
 directories Sphinx parses, how the docs are built, and how the extensions are
 used.
 
 Building the docs
 -----------------
 
-The documentation sources are found in the :file:`doc/` directory in
-the trunk.  To build the users guide in html format, cd into
-:file:`doc/` and do:
+The documentation sources are found in the :file:`doc/` directory in the trunk.
```

Comment:
This part will be obsoleted by https://github.com/matplotlib/matplotlib/pull/9513/files...